include/libcyphal/application/node/get_info.hpp

```diff
@@ -0,0 +1,129 @@
+/// @copyright
+/// Copyright (C) OpenCyphal Development Team  <opencyphal.org>
+/// Copyright Amazon.com Inc. or its affiliates.
+/// SPDX-License-Identifier: MIT
+
+#ifndef LIBCYPHAL_APPLICATION_NODE_GETINFO_HPP_INCLUDED
+#define LIBCYPHAL_APPLICATION_NODE_GETINFO_HPP_INCLUDED
+
+#include "libcyphal/presentation/presentation.hpp"
+#include "libcyphal/presentation/server.hpp"
+#include "libcyphal/types.hpp"
+
+#include <cetl/pf17/cetlpf.hpp>
+
+#include <uavcan/node/GetInfo_1_0.hpp>
+
+#include <chrono>
+#include <utility>
+
+namespace libcyphal
+{
+namespace application
+{
+namespace node
+{
+
+/// @brief Defines 'GetInfo' component for the application node.
+///
+/// No Sonar cpp:S4963 'The "Rule-of-Zero" should be followed'
+/// b/c we do directly handle resources here (namely capturing of `this` in the request callback).
+///
+class GetInfo final  // NOSONAR cpp:S4963
```

Comment:
Like Heartbeat, this one should be like `GetInfoServer` or `NodeInfoServer` or something else a bit more descriptive.